app/services/follow_service.rb

```diff
@@ -3,18 +3,20 @@ class FollowService
   class << self 
     def follow(entity, user, reason)
 
-      return unless user && is_tester(user)
+      return unless user
 
       return if (reason == 'created' || reason == 'contributed') && !should_auto_follow(entity, user, reason)
 
       follow = Follow.where(followed: entity, user: user).first_or_create
+      follow.update_attribute('muted', false)
```

Comment:
This ensures a muted follow flips back to normal when you manually follow the entity

Reply:
this should use follow.update, rather than update_attribute, so validations are run. then we need to check if it succeeded and handle validation errors